### ISSUE TITLE
error: MODULE_NOT_FOUND

### ISSUE THREAD

**Issue by DarkHeros09** [dotenvx/dotenvx#276]
Hi

I'm getting this error on 1.1.0 using `curl -sfS https://dotenvx.sh | sh`

```dotenvx run -f ./.env.test -- go test -v -cover -timeout 1m -shuffle on -count=1 ./...
node:internal/modules/cjs/loader:936
  throw err;
  ^
Error: Cannot find module '/home/runner/work/CShop/CShop/run'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function._resolveFilename (pkg/prelude/bootstrap.js:1951:46)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.runMain (pkg/prelude/bootstrap.js:1979:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```

**Comment by motdotla** [comment]
taking a look

**Comment by motdotla** [comment]
i'm unable to duplicate at first try on `mac` and `linux`

what os and arch you are on? `uname`, `uname -m`

also can you tell me what `node -v` returns for you?

**Comment by motdotla** [comment]
also is this error occurring for you when you run the install or after you've installed it and using `dotenvx run` ?

**Comment by DarkHeros09** [comment]
i get this error when i run both my dagger test, and github action ci with linux enviroment.
everything was running ok before.

my ci file
```yml
name: Run unit tests

on:
  push:
    branches: [ main ]
  pull_request:
    branches: [ main ]

jobs:

  test:
    name: Test
    runs-on: ubuntu-latest
    environment: Testing

    services:

      postgres:
        image: postgres
        env:
          POSTGRES_USER: postgres
          POSTGRES_PASSWORD: secret
          POSTGRES_DB: cdb
        ports: 
          - 6666:5432
        options: >-
          --health-cmd pg_isready
          --health-interval 10s
          --health-timeout 5s
          --health-retries 5 

    steps:
    - uses: actions/checkout@v4

    - name: Setup Go
      uses: actions/setup-go@v5
      with:
        go-version: '>=1.22.3'
        cache: false
  
    - name: Install golang-migrate
      run: |
        curl -L https://github.com/golang-migrate/migrate/releases/download/v4.17.1/migrate.linux-amd64.tar.gz | tar xvz
        sudo mv migrate /usr/bin/
        which migrate
        
    - name: Run Migrations
      run: make migrate_up
        
    - name: install dotenvx
      run: curl -sfS https://dotenvx.sh/install.sh | sh

    - name: Test
      run: dotenvx run -f .env.test -- make test
      env:
        DOTENV_PRIVATE_KEY_TEST: ${{ secrets.DOTENV_PRIVATE_KEY_TEST }}
```

**Comment by motdotla** [comment]
can you change your install script to `curl -sFS https://dotenvx.sh/?version=1.0.1 | sh` temporarily to see if that fixes things.

meanwhile, i'm still trying to repeat the issue. we added typescript types in `1.1.0` so might be related to that somehow.

**Comment by motdotla** [comment]
also, looks like you are using `go`. are you able to successfully run the go example:

```
$ echo "HELLO=World" > .env
$ echo 'package main; import ("fmt"; "os"); func main() { fmt.Printf("Hello %s\n", os.Getenv("HELLO")) }' > main.go

$ dotenvx run -- go run main.go
Hello World
```

can you share some of the contents of your `make test` command - redact anything sensitive

**Comment by DarkHeros09** [comment]
the content of make test
 `dotenvx run -f $(ENVFILE) -- go test -v -cover -timeout 1m -shuffle on -count=1 ./...`

**Comment by DarkHeros09** [comment]
> can you change your install script to `curl -sFS https://dotenvx.sh/?version=1.0.1 | sh` temporarily to see if that fixes things.
> 
> meanwhile, i'm still trying to repeat the issue. we added typescript types in `1.1.0` so might be related to that somehow.

it didn't work, i even tried both 0.45.0 and 1.0.0 to no avail.

the test pass find on my local win11 machine.

**Comment by motdotla** [comment]
ah, i see. that means your script is effectively trying to wrap dotenvx with dotenvx:

```
dotenvx run -f .env.test -- dotenvx run -f $(ENVFILE) -- go test -v -cover -timeout 1m -shuffle on -count=1 ./...
```

change to this:

```
    - name: Test
      run: make test
      env:
        ENVFILE: '.env.test'
        DOTENV_PRIVATE_KEY_TEST: ${{ secrets.DOTENV_PRIVATE_KEY_TEST }}
```

that should do the trick.

thanks for surfacing this though. the error should be more helpful when this happens.

**Comment by DarkHeros09** [comment]
ahh of course, how did i not noticed this.
I've must modified make test command, and i forgot about it.

my bad.
thank you for your help.

**Comment by motdotla** [comment]
this one was tricky to track down. fix is here and will go out as part of `1.2.0` release:

https://github.com/dotenvx/dotenvx/pull/279
